Templates/DefaultProject/Template/ExportScripts/export_standalone_monolithic.py

```diff
@@ -0,0 +1,321 @@
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+
+import argparse
+import glob
+import logging
+import os
+import o3de.export_project as exp
+import pathlib
+import platform
+import shutil
+import sys
+import time
+import psutil
+
+from o3de.export_project import process_command
+from o3de import manifest
+from typing import List
+
+
+# This script is meant to be run via o3de cli export-project command, not from Python directly!
+# This export script operates in a project-centric manner, meaning all build artifacts are centered around the project folder. 
+# In order to use this script, the command should look like the following (on windows use o3de.bat, on linux use o3de.sh):
+
+# cd path\to\o3de\engine
+# .\scripts\o3de.bat  export-project --export-scripts path\to\o3de\ExportScripts\export_standalone_monolithic_project_centric.py \
+# --project-path path\to\project --log-level INFO --output-path path\to\output --build-non-mono-tools --config release --archive-output zip \
+# --seedlist path\to\seedlist1.seed --seedlist path\to\seedlist2.seed <etc..> 
+
+# The following parameters are optional, but may be helpful:
+#   seedlist: an O3DE seedlist for describing what assets of the project should be bundled
+#   gem-folder-name-to-copy: A Gem Folder from <ProjectRoot>/Gems that should be copied to the output location of the exported project
```

Comment:
Not all projects have a Gems folder.
There are projects that have a `<ProjectRoot>/Gem` folder, `<ProjectRoot>/Code` folder or even no folder at all.

Reply:
Why do we need to copy Gems folder?

Reply:
Only some projects do. MPS for instance has data in the AWSCore Gems folder, which was what necessitated the argument to be made. It's an optional parameter, so it can be omitted if that project doesnt use it.

Reply:
Should the data be part of the Asset Bundler Seed List then?

Reply:
I think in those cases you can just pass in the seed list files manually using the `seedlist` parameter